### ISSUE TITLE
Crash on opening a different directory while another directory is loading

### ISSUE THREAD

**Issue by PeyaPeyaPeyang** [files-community/Files#7514]
### Description

If I open another directory while the directory is loading (circular progress is running), Files crashes after about 10 seconds.  
When this crash occurs, the next Files instance opened will also crash after freezing, and will not work.  
This crash can be cured by restarting the computer, but it will happen again if you perform the same operation again.  

Also, after this bug occurs, Files will freeze for about 5 minutes, but then it will work normally until the bug occurs again.
Closing a tab that is loading will also cause this bug.

### Steps To Reproduce

1. Launch Files
2. Open any directory and repeat until you get a circular progress of Loading.
3. Open a different directory.
4. Files freezes and then crashes.

### Expected behavior

Files can be used normally without crashing.

### Files Version

Version: 2.0.34.0 OS Version: 10.0.19044.1415

### Windows Version

Windows 10 21H2 19044.1415

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/7836177/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/7836178/debug_fulltrust.log)

**Comment by yaichenbaum** [comment]
@peyang-Celeron is this still an issue in the preview version? https://files.community/download/preview

**Comment by peyang-Celeron** [comment]
Thanks for your response. (Please let me know if there are any expressions that are difficult to convey because of using a translator.)

---

Yes, it is.
Also, the previously tested version froze only once every few dozen times, while the preview version freezes much more frequently  and logs several exceptions.

Files Preview Version; `2.2.5.0` 

Logs:
[debug.log](https://github.com/files-community/Files/files/8676129/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/8676135/debug_fulltrust.log)

**Comment by yaichenbaum** [comment]
We made some changes that may have resolved this, is it still an issue in the preview version? https://files.community/download/preview

**Comment by PeyaPeyaPeyang** [comment]
Thanks for the triage.
It may be my problem, but I just wanted to share that I got the error below on the preview install.
![image](https://user-images.githubusercontent.com/42040068/199419527-173d5fba-363b-4b2e-ab48-737489918bf6.png)
(Translate: `Could not install app. Error Message: Error 0xC00CEE01: XML in .appinstaller file is invalid: line 31, column 28, reason: unexpected end of input. (0xc00cee01)`)

I have successfully installed Preview itself(used to .msixbundle) manually and I will check the fix now.
